docs/understanding-tracking-design/create-tracking-plan-with-tracking-scenarios/managing-tracking-scenarios-console/index.md

```diff
@@ -0,0 +1,166 @@
+---
+
+
+## Creating a New Tracking Scenario using Console
+
+Tracking Scenarios are directly linked to Data Structures and represent the events that are documented and tracked using a certain Data Structure.
+
+To create a new Tracking Scenario, navigate to the "Data Structures" option in the menu. From there, select a data structure from the list.
+
+:::info
+Note that Tracking Scenarios can only be created from Data Structures that have been set as "Event."
+:::
+
+After selecting a data structure, click on the "Tracking Scenarios" tab, as shown in the image.
+
+![](images/ts-1.png)
+
+Next, click the 'Add new scenario' button. A new dialog box will appear with fields for basic information that can be defined during creation, as shown.
+
+![](images/ts-2.png)
+
+Tracking scenarios should be created in association with the most recent version of the data structure deployed in production. This will determine the rules for tracking implementation that can be set in the validation part of the tracking scenarios.
+
+It is always guaranteed that your tracking scenario will have rules compatible with the associated data structure version. This ensures that any event tracked by following the rules defined in the tracking scenarios will pass validation for the associated data structure.
+
+However, if a newer version of the data structure is subsequently deployed in production, you may wish to "upgrade" the tracking scenario and tracking to the newer data structure and verify that your implementation rules still meet validation.
+
+The fields available for filling at this stage are:
+
+- A text field for the **scenario name**
+- A tag field to indicate which applications this scenario is designed to track.
+- A free text field for a **description** of the tracking scenario.
+
+![](images/ts-3.png)
+
+To create a tracking scenario, click the **'Save'** button. If you want to create multiple scenarios at once, such as different scenarios for mobile and web tracking, click the **'Save & add another'** button. You can cancel the creation process by clicking the **'Cancel'** button.
+
+After creating your first tracking scenario, a list of tracking scenarios will appear on the page.
+
+![](images/ts-4.png)
+
+Until published, the scenario status is set to "Draft" to represent that it is a work in progress. During this time, you may wish to share it with others in the business to get feedback or approval. To do this, select the link to the tracking scenario and click the "Share" button located at the top right of the page. Please note that others will need access to the Snowplow Console Data Structures section to view the scenario.
+
+* * *
+
+## Adding more details and editing a Tracking Scenario
+
+To add more information or modify a tracking scenario, select it from the appropriate list view in the "Data Structure and Tracking Scenarios" section. This will open a new page displaying the details of the selected tracking scenario.
+
+![](images/ts-5.png)
```

Comment:
We need to change this screenshot one this [ticket](https://snplow.atlassian.net/browse/XP-1752) is fixed.

Reply:
Updated